feature-extensions/apigee/README.md

```diff
@@ -0,0 +1,62 @@
+## Summary
+- **Make sure you have an Apigee account with at least two environments**
+- **This installer will use whatever AWS credentials you have configured by running 'aws configure'**
+- **Please make sure you are using the same AWS credentials you used to install your Jazz deployment**
+
+## USAGE
+usage: setup.py [-h] [--aws-region AWS_REGION]
+                [--jazz-stackprefix JAZZ_STACKPREFIX] [--scm-repo SCM_REPO]
+                [--scm-username SCM_USERNAME] [--scm-password SCM_PASSWORD]
+                [--scm-pathext SCM_PATHEXT] [--jenkins-url JENKINS_URL]
+                [--jenkins-username JENKINS_USERNAME]
+                [--jenkins-password JENKINS_PASSWORD]
+                {install,uninstall} ... apigee_host apigee_org apigee_prod_env
+                apigee_dev_env apigee_svc_prod_host apigee_svc_dev_host
+                apigee_username apigee_password
+
+
+- **Even though the Install/Uninstall is possible with Parameterized approach, it will ask
+    user raw inputs for the missed parameters**
+
+#### INSTALL #####
```

Comment:
=>  ## Install

=>  ### With all the parameters
.... 
=>  ### With missing parameters
....

=>  ## Uninstall

=>  ### With all the parameters
.... 
=>  ### With missing parameters
....

Reply:
Done